Automattic/jetpack
# Carousel: ensure we do not try to use wpcom when not available

#### Changes proposed in this Pull Request:

﻿Follow-up from 1e863d73e8432e44c1820e570183e60901247c48

Let's avoid using `wpcom` when it is not available (when logged out of WordPress.com and no notifications are loaded for the admin bar for example)

#### Jetpack product discussion

* This was originally reported in the forums: https://wordpress.org/support/topic/carousel-doesnt-work-in-jetpack-9-8/

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* When logged out of your site, visit a post with a gallery.
* Click on an image to trigger the Carousel view
* You should not get any JS errors.
